CONTRIBUTING.adoc

```diff
@@ -0,0 +1,90 @@
+
+We understand that contributors cannot devote 100% of their time to a project, just as our maintainers cannot devote 100% of their time to the project.. We want to squash bugs as often as we can, but sometimes require comments from the submitter or community in order to proceed.
+
+If a bug report has been inactive (waiting on comment from submitter or community) for 14 days, a maintainer may label the issue as stale.  Once a bug report has been inactive for more than 28 days, the issue will be administratively closed. Issues can be re-opened after being administratively closed by commenting on the issue requesting to re-open and providing the necessary comment.
+
+== Enhancements
+Like bug reports, enhancements are submitted through the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page.
+
+As with Bug Reports, please do a search of the open requests first before submitting a new one to avoid duplicates. If you find an existing one, give it a +1.
+
+[NOTE]
+If this is an enhancement you intend to implement, please be sure to submit the enhancement *before* working on any code changes. This will allow members on the workshop team to have a discussion with you to ensure that it's the right design and that it makes sense to include in the workshop.
+
+Enhancements are labeled with link:https://github.com/aws-samples/aws-workshop-for-kubernetes/labels/Enhancement[Enhancement].
+
+=== Submitting an Enhancement
+Open an link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[issue] with the following:
+
+* A short, descriptive title. Ideally, other community members should be able to get a good idea of the enhancement just from reading the title.
+* A detailed description of the the proposed enhancement. Include justification for why it should be added to the workshop, and possibly example code to illustrate how it should work.
+* link:https://guides.github.com/features/mastering-markdown/[Markdown] formatting as appropriate to make the request easier to read.
+* If you intend to implement this enhancement, indicate that you'd like to the issue to be assigned to you.
+
+== Contributions
+Contributions to the workshop are done through link:https://github.com/aws-samples/aws-workshop-for-kubernetes/pulls[Pull Requests]. Please keep the following in mind when considering a contribution:
+
+* The workshop is released under the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/blob/master/LICENSE[Apache 2.0 License]. Any code you submit will be released under this license. If you are contributing a large/substantial enhancement, you may be asked to sign a link:https://github.com/aws/aws-cla[Contributor License Agreement (CLA)].
+
+* For anything but very small or quick changes, you should always start by checking the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page to see if the work is already being done by another person.
```

Comment:
"Each PR must be accompanied by an issue or enhancement" is an important requirement and would streamline the process.

Reply:
Fixed in future commit.